docs/pages/debugging/tools.mdx

```diff
@@ -60,7 +60,7 @@ The Console tab gives you access to an interactive terminal, connected directly
 
 The **Network tab** gives you insights into the network requests made by your app. You can inspect each request and response by clicking on them. This includes `fetch` requests, external loaded media, and in some cases, even requests made by native modules.
 
-This feature is only available when using the dev client and not when using Expo Go.
+> **warning** This feature is only available from Expo SDK 49+ when using the dev client. It is not available when using Expo Go.
```

Comment:
```suggestion
> **warning** This feature is only available from Expo SDK 49 and higher when using the dev client. It is not available when using Expo Go.
```

Suggestion to use [symbols as words](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#symbols-as-words)